draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.tsx

```diff
@@ -76,13 +78,19 @@ class GuidanceBlock extends React.Component<Props, State> {
       img,
       text,
       persistent,
+      prominent,
       withActionButtonArrow,
     } = this.props
 
     return (
       <div
         className={this.bannerClassName()}
-        style={{ marginTop: this.marginTop() }}
+        style={{
+          marginTop: this.marginTop(),
+          backgroundColor: prominent
+            ? colorTokens.kz.color.cluny[100]
+            : colorTokens.kz.color.white,
+        }}
```

Comment:
For styles that don't require the javascript runtime, we typically put them in Sass. This helps us avoid using inline styles. Something like this is preferred: 
```
className={classnames(this.bannerClassName(), {
  [styles.prominent]: variant === "prominent"
}}
```